deliverable/1/process.md

```diff
@@ -0,0 +1,106 @@
+- Where possible, tasks will immediately be broken down into smaller components (no subtasks for "final" tasks).
+- Tasks may be added any time, but must be estimated before being assigned.
+- Estimations will be done in terms of work hours and revised as we see fit, by majority vote.
+- Anything that will use (a) group members' time for this project should be tracked on Trello. Post-hoc estimation is better than none.
+- Each task will be a single Trello card organized into one of these five columns: 
+    - **Planned**: has not been started. Reassignment is possible.
+    - **Blocked**: depends on another task being completed first. Blocking task should be linked on blocked card.
+    - **In Progress**: may have uncommitted work by the group member(s) to which it is assigned. Frequent contact required.
+    - **Awaiting Review**: is considered completed by the last assigned group member and awaits the review of another.
+    - **Completed**: is no longer active (finished or abandoned) and is kept mostly for progress tracking.
+
+## Peer Review
+
+All code will be peer reviewed before being merged into the `master` branch. 
+
+Each peer review can be completed remotely and simply entails:
+- A critical reading of the new/modified code/commit
+- Completion of the peer review check list [here](./peer_review.md)
+- Attachment of the completed check list to the PR
+- If satisfactory: approval of the PR into master
+- Else: a comment left on the PR with needed/suggested changes
+- Appropriate updates to the Trello board
+
+## Test Driven Development
+
+Our process will incorporate most of the standard paradigms of TDD. We will write tests that initially fail in order to identify either the feature or bug that is needed. We will begin development in earnest only once the tests that have been written fully evaluate the correctness of the portion in question. This does not have to be a whole task: a good size portion might be a single method or function.
+
+## Continuous Integration
+
+All branches will be built and run with the Travis and Circle Continuous Integration tools. For both tools, configuration files and instructions for activating the GitHub add-ons are provided by the Matplotlib development team.
+
+Although they (the matplotlib developers) do not require or even strongly recommend that contributors set up CI on their own forks, we would like to have the best chance possible of having our PR merged into the master matplotlib repo. For that to be the case, we would like to know if any changes we make cause the build to fail, immediately. Their current recommended strategy involves kicking back PRs that break the build, but due to our time constraint we find this unacceptable.
+
+## Branching
+
+Our branching strategy is simple enough: where possible, work on a unique branch for your task, named after the task. 
```

Comment:
Somewhere in this section (not sure about under Branching) we should talk about re basing or other methods to simplify the commit history.

Reply:
Added a description for rebasing strategy under "Merging to master" section.